src/components/elements/Notification/FloatNotification.tsx

```diff
@@ -37,6 +38,20 @@ const FloatNotification = () => {
     triggerBulkUpdate(newJobsData);
   };
 
+  const AbortJobUuid = (uuid: string) => {
+    const job: DelayedJobData[] = [
+      {
+        uuid: uuid,
+        type: "delayedJobs",
+        attributes: {
+          isAcknowledged: true
+        }
+      }
+    ];
+    JobsSlice.setAbortJob(true);
+    triggerBulkUpdate(job);
+  };
```

Comment:
This method doesn't seem to have any dependencies within the component, and is pure with regard to the component state. It's usually best to define the function outside of the component body in cases like that. Also, please reserve upper first names for classes - this should be named `abortJobUuid`, or simply `abortJob`.